src/SmartComponents/SystemInventory/SystemInventoryCard.js

```diff
@@ -1,26 +1,40 @@
 import React from 'react';
 import { TemplateCard, TemplateCardBody, TemplateCardHeader } from '../../PresentationalComponents/Template/TemplateCard';
 import { NumberDescription } from '../../../../insights-dashboard/src/PresentationalComponents/NumberDescription/NumberDescription';
+import { IconInline } from '../../PresentationalComponents/IconInline/IconInline';
+import { injectIntl } from 'react-intl';
+import messages from '../../Messages';
+import PropTypes from 'prop-types';
 
 /**
  * System inventory card for showing system inventory and status.
  */
-const SystemInventoryCard = () => {
+const SystemInventoryCard = ({ intl }) => {
```

Comment:
No need to use HOC `InjectIntl`, I would rather see usage of `useIntl` so we can get rid of `PropTypes` and such. [ormatjs/react-intl#new-useintl-hook-as-an-alternative-of-injectintl-hoc](https://github.com/formatjs/react-intl/blob/master/docs/Upgrade-Guide.md#new-useintl-hook-as-an-alternative-of-injectintl-hoc)

Reply:
ok, updated all components to use `useIntl` now